src/app/screens/App/screens/Home/components/SearchBox.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./SearchBox.css";
+
+export default function SearchBox(props) {
+  return (
+    <div className="search-container">
+      <input
+        type="text"
+        name="searchText"
+        onChange={props.onChange}
+        placeholder="Buscar..."
+        className="search"
+      />
+      <img
+        src="assets/ASSETS/search.svg"
```

Comment:
you should import the assets at the top of the file

Reply:
`import Icon from 'path....svg'`
and then use `Icon` on `src` attribute 
`src={Icon}`